sse2neon.h

```diff
@@ -8873,13 +8873,48 @@ static inline int _sse2neon_sido_negative(int res, int lb, int imm8, int bound)
 }
 
 #define SSE2NEON_MIN(x, y) (x) < (y) ? (x) : (y)
-#define SSE2NEON_GET_LENGTH_OR_BOUND(la, lb, bound) \
-    int tmp1 = la ^ (la >> 31);                     \
-    la = tmp1 - (la >> 31);                         \
-    int tmp2 = lb ^ (lb >> 31);                     \
-    lb = tmp2 - (lb >> 31);                         \
-    la = SSE2NEON_MIN(la, bound);                   \
-    lb = SSE2NEON_MIN(lb, bound);
+
+#define SSE2NEON_CMPESTRX_GET_UPPER_BOUND \
```

Comment:
Would you modify the function body of `_mm_cmpestri` and `_mm_cmpestrm` as well?

Reply:
The `SSE2NEON_CMPESTRX_GET_UPPER_BOUND` macro is used for getting upper-bound of byte/word. What's more, `_mm_cmpestrs` and `_mm_cmpestrz` need the upper-bound information to decide whether the input string hits upper-bound or not.